docs/osf-trivia-list/dasharo.md

```diff
@@ -124,3 +124,33 @@ We want to work on changing the above paradigms or at least improve the
 state-of-the-art relation in the community to the level where threats will have
 a reasonably small impact. We believe that open-source firmware is a critical
 tool, which should be used consciously to ensure privacy and liberty.
+
+## What is open-source firmware distribution?
+
+Dasharo is 3mdeb's firmware distribution and all its components are
+open-source. We provide releases in binary form. As you know in most cases on
+x86 for firmware to be useful it has to cooperate with closed blobs. In all
+binary releases we are making sure we provide information where all components
+are coming from.
+
+Dasharo works without blobs on platforms that allow that. When we saying
+Dasharo open-source firmware distribution we mean code that is delivered by
+3mdeb that is open-source. We have no influence on the code provided by 3rd
+parties (e.g. FSP, ME, GbE etc.).
+
+In coreboot community there was some controversy about calling Dasharo
+open-source firmware distribution (for details please check [gerrit review][gerrit-review].
+We respect coreboot community opinion, so we agreed that in case of coreboot
+documentation it would be better to use open-source based firmware
+distribution. It doesn't mean we agree with that decision:
+
+* Definitions and rules used in coreboot documentation review
+* Rules seem not to be applied equally to all contributors of
+  [coreboot distribution][cb-distro]
+
+## Why Dasharo is not called coreboot firmware distribution?
+
+Dasharo is not coreboot firmware distribution because we also may imagine
```

Comment:
My proposal to modyfy this paragraph:

```
While the coreboot is now the default open-source framework for Dasharo, we do
not want to limit Dasharo to one framework. We also expect another firmware
frameworks to be a base for Dasharo, such as
[U-Boot](https://u-boot.readthedocs.io/en/latest/),
[oreboot](https://github.com/oreboot/oreboot),
[Slim Bootloader](https://slimbootloader.github.io/), or pure
[EDK II](https://github.com/tianocore/edk2).

Moreover, coreboot is not enough in most cases for booting modern computer.
Most Dasharo flavors are currently based on coreboot with EDK II payload, but
we also have
[coreboot with skiboot/heads](https://docs.dasharo.com/variants/talos_2/releases/)
payload, and we expect more flavors to appear in the future.
```

Reply:
Applied: https://github.com/Dasharo/docs/pull/503/commits/7b78e511dbd136912c4ea3674393d1db74310968